content/md/products-super-power/enrich-products-variants.md

```diff
@@ -38,25 +38,59 @@ If the background of the `Variant products` is:
 
 ## Filter on your attributes
 
-To easily manage the products with variants and their until 3 levels of enrichment, we have added a **smart** search in the grid for products with variants to automatically display what you are looking for.
+To easily manage the products with variants up to 3 levels of enrichment, we added a **smart** search in the grid for products with variants to automatically display what you are looking for. Either a product model, or a product.
 
-By default, the product models are displayed in the grid.
-For example, you have a model of t-shirt which vary by color (blue, red, grey) and by size (S, M, L, XL).
+:::info
+By default, if no filter in the grid, the products are grouped by product models, so the product models are displayed in the grid.
+:::
+
+Let's take an example with a t-shirt model "model-tshirt-divided" available in 3 colors (blue, red, grey) and 4 different sizes (S, M, L, XL).
+When you open the grid, one line for this t-shirt model is displayed (12 variant products are grouped in this product model).
 
 ![Grid with product models](../img/Products_ProductModel_Search1.png)
 
-As done on e-commerce websites, if you are looking for clothing with the color "red", the red t-shirt will be automatically displayed in the grid.
+As done on e-commerce websites, if you are looking for clothes with the color "red", red clothes will be automatically displayed in the grid.
+
+So in our example, you add the filter on the attribute `color` and select the option `red`, the product model displayed in the grid is now the "model-tshirt-divided-red" (with 4 red variant products in S, M, L, XL sizes)
```

Comment:
"the prduct displayed in the grid"?

Reply:
unless color is at the product model level?

Reply:
yes color is at the product model level, but I will change for sub product model with link :)